metadata/reportssql/drug_functions.sql

```diff
@@ -894,6 +957,43 @@ BEGIN
 END$$
 DELIMITER ;
 
+-- concatenateARTDrugs
+
+DROP FUNCTION IF EXISTS concatenateARTDrugs;
+
+DELIMITER $$
+CREATE FUNCTION concatenateARTDrugs(
+    p_patientId INT(11),
+    p_startDate DATE,
+    p_endDate DATE) RETURNS TEXT
+    DETERMINISTIC
+BEGIN
+    DECLARE result TEXT;
+    
+    SELECT GROUP_CONCAT(d.name SEPARATOR ";") INTO result
+    FROM orders o
+        JOIN drug_order do ON do.order_id = o.order_id
+        JOIN concept c ON do.duration_units = c.concept_id AND c.retired = 0
+        JOIN drug d ON d.drug_id = do.drug_inventory_id AND d.retired = 0
+    WHERE o.patient_id = p_patientId AND o.voided = 0
+        AND drugIsARV(d.concept_id)
+        AND o.order_action <> "DISCONTINUE"
+        AND o.date_stopped IS NULL
+        AND treatmentIsWithinReportingPeriod(
+            p_startDate,
+            p_endDate,
+            o.scheduled_date,
+            calculateTreatmentEndDate(
+                o.scheduled_date,
+                do.duration,
+                c.uuid)
+            )
+    ORDER BY calculateDurationInDays(o.scheduled_date,do.duration,c.uuid) ASC,
```

Comment:
@tresorluz we won't need `ORDER BY` in this case, the result is being grouped and concatenated. Also, removing this will help improve the report performance because ORDER BY calling a function is an expensive operation